### ISSUE TITLE
no such file or directory error when executing "source /opt/ros/galactic/setup.bash"

### ISSUE THREAD

**Issue by YuRuiii** [MiRoboticsLab/cyberdog_sim#4]
I'm on Ubuntu 20.04 and I tried to set up the environment for cyberdog_sim, but it failed when I executed the following command in the folder `cyberdog_sim`:

```shell
source /opt/ros/galactic/setup.bash
```

The log result is: `no such file or directory: /home/xx/cyberdog_sim/setup.sh`

I checked `/opt/ros/galactic/setup.bash` and `cyberdog_sim/cyberdog_sim.repos`, and I think maybe no `setup.sh` will be generated in the folder `cyberdog_sim`.

Could you please kindly give me some advice on how to solve the problem? Many thanks!

**Comment by Alkrick** [comment]
Hello, have you followed the installation and compile instructions on the readme page?
Could you please provide a detailed description of your installation process and where exactly the error occurs?

It seems that ros-galactic was not installed properly on your system

**Comment by YuRuiii** [comment]
Hi Alkrick! Thank you so much for your reply (and for the tips you mentioned in issue #3). I failed to get any notifications for your reply so I replied late. Sorry about that.
I found out that the problem was 
I solved the problem following https://stackoverflow.com/questions/54452054/ros-installation-no-such-file-or-directory. It was because I was on `zsh` instead of `bash`, and `zsh` interprets the file in a different way. 
Thank you so much!
